web/src/main/resources/assets/js/main-template.js

```diff
@@ -239,6 +252,17 @@ $(document).ready(function (e) {
     checkInput();
 });
 
+function profileDisplayName(profileName) {
+   return translate.tr(profileName) + ' (' + profileName + ')'
```

Comment:
Any better idea? For profile names like `my_special_super_car` we will get something like "web.my_special_super_car (my_special_super_car)"...

Reply:
Initially I was concerned that some non-standard custom names cannot be translated, but you are right probably just one of the two is enough. If we remove the info in brackets we would get something like `web.my_special_car` for non-standard names, and something like `Auto` for standard-names like `car`. This could be slightly improved by removing the `web.` for non-standard names. Is this what you mean?

Reply:
Yes

Reply:
Cool yes this is better, done.